factorial.js

```diff
@@ -1,6 +1,15 @@
 function factorial(num) {
+  console.log('factorial, num is ' + num)
   // my awesome code
-  return 0;
+  if(num > 5) {
```

Comment:
I'm not sure what this has to do with calculating a factorial; could you explain?